include/tins/ip_reassembler.h

```diff
@@ -190,7 +190,6 @@ class TINS_API IPv4Reassembler {
     address_pair make_address_pair(IPv4Address addr1, IPv4Address addr2) const;
     
     streams_type streams_;
-    OverlappingTechnique technique_;
```

Comment:
This one can't be removed actually. I know it's not being used but it was added so that eventually (potentially) the code can handle IP reassembly using different techniques. So this warning we'll have to cope with (plus, this should only be a warning when building libtins itself).